src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,42 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+        Figure[] figures = new Figure[(randomNumber(1, 100))];
```

Comment:
100 and 4 are magic numbers, let's make them constants

Reply:
done